scripts/gitignore_remove_generated_files.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -eu
+
+GITIGNORES=$(find . -name ".gitignore")
+
+for GITIGNORE in $GITIGNORES; do
```

Comment:
Similarly to above we can remove the comments using sed

`sed -i '/###START_COMMENTED_GENERATED_FILES###/,/###END_COMMENTED_GENERATED_FILES###/s/^#//'`